Rakefile

```diff
@@ -24,7 +24,7 @@ end
 
 desc "Run RSpec tests for TravisCI that don't require WinRM"
 RSpec::Core::RakeTask.new(:travisci) do |t|
-  t.rspec_opts = '--tag ~winrm --tag ~appveyor_agents --tag ~puppetserver'
+  t.rspec_opts = '--tag ~winrm --tag ~appveyor_agents --tag ~puppetserver --tag ~powershell'
```

Comment:
Are you running the powershell tests on linux?

Reply:
My suggestion was the opposite. We might support powershell on linux, and there's probably nothing windows-specific about it. https://github.com/puppetlabs/bolt/pull/775#discussion_r238873814 specifically.

Reply:
I have not been running the powershell tests on my laptop. In fact I had not even really considered that as an option for linux. Will explore how this should work more in depth.

The following work with PR as is.
```
cas@cas-ThinkPad-T460p:~/working_dir/bolt$ bolt command run "powershell -noprofile -noninteractive -command 'Write-host hi'" -n localhost
Started on localhost...
Finished on localhost:
  STDOUT:
    hi
Successful on 1 node: localhost
Ran on 1 node in 0.89 seconds
```
```
cas@cas-ThinkPad-T460p:~/working_dir/bolt$ cat argv.ps1 
#!/usr/bin/powershell
Write-Host $args
cas@cas-ThinkPad-T460p:~/working_dir/bolt$ bolt script run argv.ps1 one -n localhost
Started on localhost...
Finished on localhost:
  STDOUT:
    one
Successful on 1 node: localhost
Ran on 1 node in 1.21 seconds
```